myosuite/envs/myo/assets/hand/MyoHand_tabletop.xml

```diff
@@ -37,7 +37,7 @@
         <geom type="plane" rgba=".1 .1 .1 1" size="10 10 1" pos="0 0 -.76"/>
 
         <body pos="0.1 -0.55 0.32" euler="0 0 0">
-            <joint name="ARTx" pos="0 0 0" axis="1 0 0" type="slide" range="-.05 .05" damping="20" limited="true"/>
+            <joint name="ARTx" pos="0 0 0" axis="1 0 0" type="slide" range="-.25 .25" damping="20" limited="true"/>
```

Comment:
@Vittorio-Caggiano  -- IIUC you wanted to reduce this range, not expand it right?

Reply:
no, needed to extend the range to cover some larger movements

Reply:
Got it